drivers/clock_control/Kconfig.aspeed

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 ASPEED Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_AST10X0
+	bool "ASPEED clock control driver"
+	depends on SOC_SERIES_AST10X0
```

Comment:
change this to:

```
default y
depends on DT_HAS_ASPEED_AST10X0_CLOCK_ENABLED
```

and remove the `depends on SOC_SERIES_AST10X0`

Reply:
Thank you for your review.  Is `DT_HAS_ASPEED_AST10X0_CLOCK_ENABLED` auto-generated?  I couldn't find this in any files in `build/`.

Is it okay by using `dt_compat_enabled`?

```
DT_COMPAT_ASPEED_AST10X0_CLOCK := aspeed,ast10x0-clock

config CLOCK_CONTROL_AST10X0
	bool "ASPEED clock control driver"
	default y
	depends on $(dt_compat_enabled,$(DT_COMPAT_ASPEED_AST10X0_CLOCK))
	help
	  This option enables the clock driver for ASPEED AST10X0 series SOC.
```

Reply:
It's autogenerated.

Take a look at PR https://github.com/zephyrproject-rtos/zephyr/pull/48152 to see examples of what this should look like.

Reply:
Thank you for your comment.  I rebase my branch and force pushed.